test/ng/directive/ngOptionsSpec.js

```diff
@@ -2143,6 +2143,26 @@ describe('ngOptions', function() {
       expect(scope.selected).toEqual([]);
     });
 
+    it('should be possible to specify a value for empty options', function() {
+      var html = '<select ng-model="someModel" ng-options="c for c in choices" ng-empty-value="someVar">' +
```

Comment:
You should use the createSingleSelect fn here, to make it consistent with other tests